s3tests/functional/utils.py

```diff
@@ -42,7 +42,7 @@ def generate_random(size, part_size=5*1024*1024):
 # syncs all the regions except for the one passed in
 def region_sync_meta(targets, region):
 
-    for (k, r) in targets.iteritems():
+    for (k, r) in targets.items():
```

Comment:
probably the tuple packing can be avoided here, since we don't seem to use the tuple anyway, though mostly a nit.